packages/xrpl/test/integration/requests/serverState.test.ts

```diff
@@ -116,6 +117,8 @@ describe('server_state', function () {
         'node_size',
         'initial_sync_duration_us',
         'ports',
+        'git',
+        'network_id',
```

Comment:
Why does this need to be removed?

Reply:
The integration tests use a standalone docker instance of rippled. This rippled does not use a custom `network_id` field in its configuration. Hence, this field is not returned by `server_state` RPC.

Reply:
It is difficult to predict the `hash` and the `branch` of the `git` sub-field.

Reply:
`git` I understand, `network_id` is unnecessary.

Reply:
the tests won't pass without this ignore directive

Reply:
Alternatively, I'll have to update the rippled.cfg file

Reply:
@mvadari I'd like to discuss this issue offline. I'll merge this PR at the moment because explorer changes need the client library